pkg/notes/options.go

```diff
@@ -151,5 +159,32 @@ func (o *Options) ValidateAndFinish() error {
 		}
 	}
 
+	// TODO: It would be nice to validate the go template early (e.g. parse it).
+	// Override template path when string template is given.
+	if o.Template != "" {
+		o.TemplatePath = ""
+	}
+
+	if o.TemplatePath != "" {
+		b, err := ioutil.ReadFile(o.TemplatePath)
+		if err != nil {
+			return errors.Wrap(err, "reading template")
+		}
+
+		if len(b) == 0 {
+			return errors.Errorf("template %q is empty and template string not specified", o.TemplatePath)
+		}
+		o.Template = string(b)
+	}
+
+	if o.Template == "" && o.TemplatePath == "" {
+		o.Template = internal.DefaultReleaseNotesTemplate
+	}
+
+	// Add appropriate log filtering
+	if o.Debug {
+		logrus.SetLevel(logrus.DebugLevel)
+	}
```

Comment:
We already seem to do that on the very top of this function. Is there a reason why we do it again here?

Reply:
Acknowledged.